pages/Footer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Link from 'next/link';
+
+const Footer = () =>(
+
+  <Container fluid style={{backgroundColor: '#212529', color: '#fff'}}>
+    <Container>
+    <ul>
```

Comment:
Would it be possible to use styling from `base-nav` example https://getbootstrap.com/docs/4.6/components/navs/#base-nav